bootstrap.c

```diff
@@ -81,25 +82,86 @@ static const char *get_host_triple(void) {
     return global_buffer;
 }
 
+#if defined(__WIN32__)
+#define HOST_PATHSEP '\\'
+#else
+#define HOST_PATHSEP '/'
+#endif
+
+static const char *get_parent_dir(const char *cwd) {
+    // safe for our use case
```

Comment:
This does not document the assumption, which sounds unrelated to the use case.

Reply:
I don't think it is a problem, since the memory will be reclaimed by the kernel. I added the comment to make it clear that leaking is ok, but maybe it should be improved?

Thanks.